importer/SeFornminSv.py

```diff
@@ -145,12 +145,18 @@ def set_location(self):
             else:
                 self.add_to_report("plats", self.plats)
         if self.has_non_empty_attribute("socken"):
+            socken_dict = self.data_files["socken"]
             socken = self.get_socken(self.socken, self.landskap)
-            if socken is not None:
+            if socken:
                 self.add_statement("location", socken)
             else:
-                raw_socken = "{} ({})".format(self.socken, self.landskap)
-                self.add_to_report("socken", raw_socken)
+                try:
+                    socken = [x["item"] for x in socken_dict
+                              if x["itemLabel"].startswith(self.socken)][0]
```

Comment:
if you are going via Label I would do:
```python
socken_candidate = [x["item"] for x in socken_dict
                                  if x["itemLabel"].startswith(self.socken)]
if len(socken_candidate != 1):
    raise ValueError
self.add_statement("location", socken_candidate[0])
```
Because I don't trust that there isn't at least one case of two of them starting with the same name.